packages/react-refresh-utils/internal/helpers.ts

```diff
@@ -133,21 +139,30 @@ function shouldInvalidateReactRefreshBoundary(
 }
 
 var isUpdateScheduled: boolean = false
+// This function aggregates updates from multiple modules into a single React Refresh call.
 function scheduleUpdate() {
   if (isUpdateScheduled) {
     return
   }
 
-  function canApplyUpdate() {
-    return module.hot.status() === 'idle'
+  function canApplyUpdate(status: ModuleHotStatus) {
+    return status === 'idle'
+  }
+
+  let applyUpdate = canApplyUpdate(module.hot.status())
+
+  const statusHandler = (status) => {
```

Comment:
I'm using `let`/`const` and arrow functions here, but we're not doing it anywhere else in this file. Is this an issue?